docker/etc/nginx/custom/03_derivative_images.conf

```diff
@@ -0,0 +1,38 @@
+## Location for derivative images to avoid hitting Drupal for invalid URLs or
+## if the source image doesn't exist.
+location /sites/default/files/styles/ {
+
+  ## Allowed derivative images.
+  ## As mentioned in the main README, this needs to be updated when
+  ## new image styles or different directories to store images are added.
+  location ~ "^/sites/default/files/styles/(announcement|icon|thumbnail|small|medium|large|media_library|guideline_thumbnail)/public/(?<file_path>(images|previews)/[^?]+)" {
+    ## The following lines are copied from the apps/drupal/drupal.conf file.
+    access_log off;
+    expires 30d;
+    ## No need to bleed constant updates. Send the all shebang in one
+    ## fell swoop.
+    tcp_nodelay off;
+    ## Set the OS file cache.
+    open_file_cache max=3000 inactive=120s;
+    open_file_cache_valid 45s;
+    open_file_cache_min_uses 2;
+    open_file_cache_errors off;
```

Comment:
@cafuego This was copied from the base drupal.conf: https://github.com/UN-OCHA/docker-images/blob/294f6d052a8d8d4d4505765c466bee56bac9e9d1/php/php-k8s-v8/etc/nginx/apps/drupal/drupal.conf#L145-L158. Do you think that's still relevant as is?

Reply:
I think we can clean that up; but that's low priority. Most of those asset files would be getting cached by varnish anyway, so performance tweaks in nginx are ... not very helpful :-)